tests/swarm/test_integration_397_398.py

```
@@ -0,0 +1,325 @@
+        bus1 = integration_setup["bus1"]
+        bus2 = integration_setup["bus2"]
+        serializer = integration_setup["serializer"]
+
+        received_configs = []
+
+        def config_subscriber(msg: SwarmMessage):
+            try:
+                config = serializer.deserialize(msg.payload, SwarmConfig)
+                received_configs.append(config)
+            except Exception:
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)